src/sql-parser/src/ast/defs/ddl.rs

```diff
@@ -951,14 +951,13 @@ impl<T: AstInfo> AstDisplay for CreateSinkConnection<T> {
     fn fmt<W: fmt::Write>(&self, f: &mut AstFormatter<W>) {
         match self {
             CreateSinkConnection::Kafka {
-                broker,
+                connection,
                 topic,
                 key,
                 consistency,
             } => {
-                f.write_str("KAFKA BROKER '");
-                f.write_node(&display::escape_single_quote_string(broker));
-                f.write_str("'");
+                f.write_str("KAFKA ");
```

Comment:
`KAFKA CONNECTION`?

Reply:
The `CONNECTION` gets printed by the connection object

Reply:
oh nice!